src/mpcCoreKit.ts

```diff
@@ -872,39 +891,66 @@ export class Web3AuthMPCCoreKit implements ICoreKit, IMPCContext {
     };
   }
 
-  public async sign(
-    data: Buffer,
-    opts?: {
-      hashed?: boolean;
-      secp256k1Precompute?: Secp256k1PrecomputedClient;
-      keyTweak?: BN;
+  public async signECDSA(data: Uint8Array, opts?: { hashed?: boolean; secp256k1Precompute?: Secp256k1PrecomputedClient }) {
```

Comment:
This is confusing. Precompute function is called `precompute_secp256k1`. Sign function is called `signECDSA`. Should be consistent!?
Moreover, ECDSA could be any curve. Do you always mean ECDSA over curve secp256k1 here?